uips/uip-7.md

```diff
@@ -0,0 +1,164 @@
+| uip            | 7                                                                         |
+| -------------- | ------------------------------------------------------------------------- |
+| title          | Noble Forwarding Address Design                                           |
+| description    | Specification of generation of Penumbra-linked Noble forwarding addresses |
+| author         | Chris Czub <chris@penumbralabs.xyz>                                       |
+| discussions-to | <https://forum.penumbra.zone>                                             |
+| status         | Draft                                                                     |
+| type           | Standards Track                                                           |
```

Comment:
@hdevalence do you have opinions on whether this should be Standards Track or Informational? /cc @cronokirby